<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-SCJRFP3V" failures="1" tests="11" name="ssfp.SSFPtestingpart3" time="238.172" errors="0" timestamp="2022-09-30T15:21:45 EDT" skipped="0">
  <testcase classname="ssfp.SSFPtestingpart3" name="search17" time="31.488"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search18" time="23.711"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search20" time="24.113"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search15" time="18.663"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="items" time="13.222"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search23" time="34.330"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search21" time="27.597"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search19" time="23.196"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search22" time="14.796">
    <failure message="expected [true] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at ssfp.POMssfp.search22(POMssfp.java:446)
at ssfp.SSFPtestingpart3.search22(SSFPtestingpart3.java:61)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- search22 -->
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search16" time="14.709"/>
  <system-out/>
  <testcase classname="ssfp.SSFPtestingpart3" name="search24" time="12.347"/>
  <system-out/>
</testsuite> <!-- ssfp.SSFPtestingpart3 -->
